tests/test_flow_signal.py

```diff
@@ -1,9 +1,14 @@
+from django.conf.urls import include, url
+from django.contrib.auth.models import User
 from django.dispatch import Signal
 from django.test import TestCase
 
-from viewflow import flow
+from viewflow import flow, views
 from viewflow.base import Flow, this
 from viewflow.flow.signal import Receiver
+from viewflow.signals import task_finished
+from viewflow.test import FlowTest
+from viewflow.views import ProcessView
 
 
 class Test(TestCase):
```

Comment:
It is the unit test case for the flow.Signal. The issue that you addresses is belongs to the viewflow testing framework, so the test case should be in tests/test_tests.py